default/python/AI/AIstate.py

```diff
@@ -93,6 +94,37 @@ def __init__(self, aggression):
         self.empire_standard_enemy = (4, ((4, 1),), 0.0, 10.0)  # TODO: track on a per-empire basis
         self.empire_standard_enemy_rating = 40  # TODO: track on a per-empire basis
 
+        # Check the optionsDB for the behavior bypass values and create
+        # the character.
+        def get_behavior_bypass_value(name, default, sentinel):
+            """Fetch the bypassed behavior value or return the default"""
```

Comment:
I moved this function definition from within AIState to be a free function in Character.py.  Its definition in AIState was misplaced as it only currently deals with bypassing character behaviors.